conf/machine/pcengines-apu2.conf

```diff
@@ -0,0 +1,6 @@
+#@TYPE: Machine
```

Comment:
@Asiderr There is no point in duplicanting those files from meta-pcengines. We are cloning meta-pcengines via kas.yml and they will be used fro there